classes/module/Module.php

```diff
@@ -23,6 +23,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
+use Exception;
```

Comment:
To remove ;)

Reply:
I have no opinion on this (use `use` for native php classes) but I think @PierreRambaud asked me to do the opposite a long time ago 🤔

Reply:
It just to fixing Travis ;)

Reply:
I asked to use namespace object when it's needed (in Symfony pages for example) :sweat_smile: 
It's not mandatory to remove the use here, it can help to see which Exception are thrown in this class.

Reply:
My bad, sorry @Progi1984